service/pom.xml

```diff
@@ -291,7 +292,7 @@ under the License.
                     </execution>
                 </executions>
                 <configuration>
-                    <classifier>exec</classifier>
+                    <classifier>${spring-boot-maven-plugin.classifier}</classifier>
```

Comment:
Let implement it in a way, that library jar is built on every build with its own classifier ("lib"?), without need to specify classifier explicitly, and keep exec semantics of default jar. Otherwise we'll have semantics of artifacts changing within same classifier depending on build settings, which is very error-prone.

Reply:
I agree this way is better. I did it, please check.